packages/scaffold-ui/src/views/w3m-wallet-send-view/index.ts

```diff
@@ -69,18 +82,30 @@ export class W3mWalletSendView extends LitElement {
     this.unsubscribe.forEach(unsubscribe => unsubscribe())
   }
 
+  public override async firstUpdated() {
+    await this.handleSendParameters()
+  }
+
   // -- Render -------------------------------------------- //
   public override render() {
     this.getMessage()
 
+    const isReadOnly = Boolean(this.params)
+
+    if (!this.address) {
+      throw new Error('w3m-wallet-send-view: No account connected')
+    }
```

Comment:
we said we were not doing the route to connection screen flow now right? 
would be cool to bring it up to sam/chris